CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Added
+- `Subtable::codepoints`
```

Comment:
It should be `cmap::Subtable::codepoints`, because we also have `kern::Subtable`.